backend/models/clients.js

```diff
@@ -0,0 +1,39 @@
+const mongoose = require('mongoose')
+
+const clientSchema = new mongoose.Schema({
```

Comment:
Schema looks good! We should add extra information regarding what fields are required (expecting non-null) and default values. Please see Task 4 on the Sprint 2 page (https://www.notion.so/h4i/Sprint-2-44ae536312bf4c71a0057500cdd15ad7) for updated information on required/default field.

To include this code in mongoose, you simply need to edit the attributes as follows:
`attributeName: {type: type, required: true, default: defaultVal}`

Let me know if you run into trouble with this, and we can troubleshoot.

Reply:
Alright I updated the schema. Let me know if I forgot to add something or something needs to be changed.

Reply:
Thanks for making the changes. It's super close now, just that there should be a `required: true` on `foodDays` and `frozenDay`. Good to go aside from that.